lib/modules/webserver/index.js

```diff
@@ -50,18 +50,25 @@ class WebServer {
   registerConsoleCommands() {
     const self = this;
     self.embark.registerConsoleCommand((cmd, _options) => {
-      if (cmd === 'webserver start') {
-        self.events.request("start-webserver");
-        return " ";
-      }
-      if (cmd === 'webserver stop') {
-        self.events.request("stop-webserver");
-        return __("stopping webserver") + "...";
-      }
-      return false;
+      return {
+        match: () => cmd === "webserver start",
+        process: (callback) => {
+          self.events.request("start-webserver");
+          callback(null, "OK");
```

Comment:
Why do we add "OK"? Shouldn't the absence of error be sufficient? Or is that what's printed? Either way, printing nothing or success would be better. 
Maybe, if the var is empty, print 'Success'

Reply:
Can you make it so that it doesn't print anything?
Otherwise, is there a way to add a callback to webserver start/stop to really know if it was a success?

Reply:
I will try to see to add callback to the command